README.md

```diff
@@ -75,62 +68,50 @@ In your new issue, please provide the following details:
 
 Ideally, also check to see if the dataset already exists in the 🤗 [Hub](https://huggingface.co/datasets). 
 
-If an admin approves it, the dataset will count toward the hackathon contributions.
-
-### 2. Create a dataset repository
+If an admin approves it, then you are welcome to implement this dataset and it will count toward contribution credit.
 
-Make a repository via the 🤗 Hub [here](https://huggingface.co/new-dataset) with the following details
+### 2. Implement the dataloader for your dataset
 
-+ Set Owner: bigscience-biomedical
-+ Set Dataset name: the name of the dataset 
-+ Select Private
-+ Click `Create dataset`
+A step-by-step guide on how you can implement a data-loading script can be found [here](CONTRIBUTING.md).
 
-### 3. Clone the dataset repository
+Please ensure your dataloader follows our expected biomedical schema, found [here](#Template-scripts)
 
-Using terminal access, find a location to place your github repository. In this location, use the following command:
-
-```
-git clone https://huggingface.co/datasets/bigscience-biomedical/chemprot
-```
-### 4. Implement the dataloader for your dataset
+**Please do not upload the data directly; if you have a specific question or request, [reach out to an admin](#Community-channels)**
 
-A step-by-step guide on how you can implementing a data-loading script can be found [here](CONTRIBUTING.md).
+#### Template scripts
 
-Please ensure your dataloader follows our expected biomedical schema <!---  @Natasha This needs to be a hyperlink [Biomedical Schema]() -->
+We provide template scripts for the most common biomedical NLP schemas: <br>
 
-**Please do not upload the data directly; if you have a specific question or request, [reach out to an admin](#Community-channels)**
+1. [Named entity recognition (NER), relational extraction (RE), event extraction](templates/template_ner.py)
+2. [Translation](templates/template_translation.py)
+3. [Part-of-speech (POS)](templates/template_pos.py)
+4. [Question-answering (QA)](templates/template_qa.py)
+5. [Natural language inference (NLI)](templates/template_nli.py)
+6. [Normalization](templates/template_normalization.py)
+7. [Sentence classification](templates/template_sentcls.py)
+8. [Fact verification](templates/template_factver.py)
+9. [Semantic similarity](templates/template_semsim.py)
+10. [Topic classification](templates/template_topiccls.py)
+11. [Co-reference](templates/template_coref.py)
 
```

Comment:
@hakunanatasha Here, are we intending to direct them to template script under templates/<> for each tasks (which doesn't exist at the moment)  OR direct them to schemas? (under schema)

Reply:
I'm starting to think we don't need a template for each task? What if we just had one template and then made it a goal to implement one full example for each task type in the examples directory (and made sure there was at least one local files example)? I feel like there is a lot of boiler plate in the templates and that a lot of the differences are covered in the schemas directory and README, 
* https://github.com/bigscience-workshop/biomedical/tree/master/schemas
* https://github.com/bigscience-workshop/biomedical/blob/master/task_schemas.md

Reply:
Here's what I'm thinking:

(1) A markdown page discussing and describing all schemas
(2) Each task example pointing to the schema and 1 implemented example
(3) one UNIVERSAL template that walks through what each of the core functions of the dataloader is

Reply:
yes, that sounds like a nice format :+1: